app/Transformers/BeatmapSetTransformer.php

```diff
@@ -39,15 +40,29 @@ public function transform(BeatmapSet $beatmap = null)
             'beatmapset_id' => $beatmap->beatmapset_id,
             'title' => $beatmap->title,
             'artist' => $beatmap->artist,
+            'description' => $beatmap->description(),
```

Comment:
probably should not be part of default json since it incurs two additional queries?